pkg/scheduler/api/job_info.go

```diff
@@ -370,6 +370,10 @@ type JobInfo struct {
 	// * value means workload can use all the revocable node for during node active revocable time.
 	RevocableZone string
 	Budget        *DisruptionBudget
+
+	// The RootHypernode property of a job is the hypernode that serves as the smallest root in the hypernode tree.
+	// A job has multiple tasks, each belonging to a hypernode. This RootHypernode is the topmost and lowest common ancestor among the hypernodes of all tasks within the job.
+	RootHyperNode string
```

Comment:
When is `RootHyperNode` assigned? I didn't find any codes assign the value to `RootHyperNode`

Reply:
when the task of job commit, the hyperNode of the task will assigned to job. it will add the code later.